procurement_for_service/README.rst

```diff
@@ -0,0 +1,15 @@
+Procurement for service
+=======================
+
+With this module, confirming order, if the product included in the sales order
```

Comment:
Alfredo, tienes que tratar de simplificar la redacción en inglés de los contenidos para que se entienda correctamente:

```This module allows to generate procurements from confirmed sale orders when the product is a service.```

Recuerda también que es una explicación para usuario funcional, por lo que no hay que entrar en detalles del tipo "si es MTO o buy" aquí. Esos detalles se pueden explicar en la sección `Usage` o `Configuration` (que por cierto nunca utilizas).

Aunque no se sintetice de esa forma la finalidad del módulo, hay que seguir estas reglas para redactar en inglés:

* Evitar los gerundios. Ellos hablan atribuyendo a una persona (normalmente la segunda). En lugar de `confirming`, se pondría `when you confirm`.
* En el inglés, casi todo lleva artículo. Por tanto, debería ser `when you confirm a sales order`.
* Esto es tanto en español como en inglés, las comas deben separar pausas lógicas en las frases para ayudar al lector a la comprensión. Una coma entre `sales order line` y `is a service` está cortando entre medias una frase copulativa. Las comas suelen ponerse en determinados complementos circunstanciales (de tiempo, lugar, etc), o en atributos explicativos.